packages/conversation/src/rest/v1/conversation-domain-api.ts

```diff
@@ -83,7 +83,8 @@ export class ConversationDomainApi implements Api {
       }
       const apiClientOptions = this.buildApiClientOptions(this.sinchClientParameters);
       this.client = new ApiFetchClient(apiClientOptions);
-      this.client.apiClientOptions.basePath = this.buildBasePath(region);
+      this.client.apiClientOptions.hostname = this.sinchClientParameters.conversationHostname
```

Comment:
This logic lead to end user cannot set a domain to be used dynamically with a region value right?
I mean; setting a host-name to `https://${region}.my-own-domain.foo` then do not change the region handling from his code (say region was set to `US` or not set to use the default one) to use `https://US.my-own-domain.foo`. 
Only way to have this is to set variable `this.sinchClientParameters.conversationHostname` to `https://US.my-own-domain.foo`

Could require a note for SDK's user (_region is not used when hostname is_)

Reply:
I've considered that the user would override the hostname to replace it in full, the use case being to use a mock server or a test system, for which I don't think regions would be put in place.
I'll make sure to add it to the documentation before the GA

Reply:
Yes, use case to override hostname seems to not require handling dedicated region 👍